docs/index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>LibertyBans Document</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+  <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta name="keywords" content="doc,docs,documentation,libertybans,minecraft,plugins">
+  <meta name="description" content="A Document for LibertyBans plugin">
+  <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <!-- fontawesome -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/fontawesome.min.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/brands.min.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/regular.min.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/solid.min.css">
+  
+  <link
+  rel="stylesheet"
+  href="//cdn.jsdelivr.net/npm/docsify-print@latest/dist/style.css"
+  />
+  <!-- Theme -->
+  <link 
+    rel="stylesheet"
+    href="//cdn.jsdelivr.net/npm/docsify-darklight-theme@latest/dist/style.min.css"
+    title="docsify-darklight-theme"
+    type="text/css"
+  />
+
+</head>
+<body>
+  <div id="app">We are building somthing special for you ...</div>
+  <script>
+    window.$docsify = {
+      search: 'auto',
+      coverpage: false,
+	    loadSidebar: true,
+	    loadNavbar: true,
+	    auto2top: true,
+	    repo: 'https://github.com/A248/LibertyBans',
+      name: 'LibertyBans',
+      search: {
+        noData: {
+          '/': 'No results!'
+        },
+        paths: 'auto',
+        placeholder: {
+          '/': 'Search'
+        }
+      },
+      toc: {
+      tocMaxLevel: 5,
+      target: 'h2, h3, h4, h5, h6'
+      },
+      editLink: {
+      text: 'Edit this document',
+      // text: '<span style="color: limegreen;">Edit this document</span>',
+      repo: 'A248/LibertyBans',
+      // repo: 'https://github.com/cyrilf/docsify-edit-link/edit/master/docs/',
+      cssClass: 'docsify-edit-link m-0'
+      },
+      lastModified: {
```

Comment:
This doesn't render at the moment: https://ahdg6.github.io/LibertyBans/#/Silent-Punishments. I suspect that is because the lastModified feature is drawing information from A248/LibertyBans, but since this PR is not merged, the lastModified feature cannot retrieve the last time the document was modified.